apps/explorer/src/ui/DisclosureBox.tsx

```diff
@@ -2,37 +2,61 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Disclosure } from '@headlessui/react';
-
-import { ReactComponent as ChevronDownIcon } from './icons/chevron_down.svg';
+import { ChevronRight12 } from '@mysten/icons';
+import { cva, type VariantProps } from 'class-variance-authority';
 
 import type { ReactNode } from 'react';
 
-export type DisclosureBoxProps = {
+const disclosureBoxStyles = cva('', {
+    variants: {
+        variant: {
+            primary: 'bg-gray-40 rounded-lg',
+            outline:
+                'bg-transparent border border-gray-45 hover:bg-gray-40 hover:border-transparent rounded-2lg',
+        },
+    },
+    defaultVariants: {
+        variant: 'primary',
+    },
+});
+
+// adding a sub title to the disclosure box for disappearing preview
+export interface DisclosureBoxProps
+    extends VariantProps<typeof disclosureBoxStyles> {
     defaultOpen?: boolean;
     title: ReactNode;
+    preview?: ReactNode;
     children: ReactNode;
-};
+}
 
 export function DisclosureBox({
     defaultOpen,
     title,
     children,
+    preview,
+    variant,
 }: DisclosureBoxProps) {
     return (
-        <div className="rounded-lg bg-gray-40">
+        <div className={disclosureBoxStyles({ variant })}>
             <Disclosure defaultOpen={defaultOpen}>
-                <Disclosure.Button
-                    as="div"
-                    className="flex cursor-pointer flex-nowrap items-center py-3.75 px-5"
-                >
-                    <div className="flex-1 text-body font-semibold text-gray-90">
-                        {title}
-                    </div>
-                    <ChevronDownIcon className="-rotate-90 text-gray-75 ui-open:rotate-0" />
-                </Disclosure.Button>
-                <Disclosure.Panel className="px-5 pb-5">
-                    {children}
-                </Disclosure.Panel>
+                {({ open }) => (
+                    <>
+                        <Disclosure.Button
+                            as="div"
+                            className="flex cursor-pointer flex-nowrap items-center py-3.75 px-5"
+                        >
+                            <div className="flex flex-1 text-body font-semibold text-gray-90">
+                                {title}
+                                {preview && !open ? preview : null}
```

Comment:
Should there be a gap here? 
<img width="342" alt="Screenshot 2023-03-24 at 11 22 20 AM" src="https://user-images.githubusercontent.com/109986297/227608677-4d978642-2b6c-43f4-87d2-17eda24a6137.png">